cmd/create/accountroles/cmd.go

```diff
@@ -292,6 +292,11 @@ func run(cmd *cobra.Command, argv []string) {
 		os.Exit(1)
 	}
 
+	if args.forcePolicyCreation && managedPolicies {
```

Comment:
NP
Should we move this check to line `154`? right after determining if the user has chosen managed policies.